bin/docserv

```diff
@@ -223,10 +224,12 @@ class DocConfParser:
         if not self.config['targets'][target]['active'] == "yes":
             logger.debug("Target %s not active." % target)
             return False
-        # validate with sknorrs magic bash script
+        # validate and stitch files together with bash script
+        # FIXME: productschema: how do I define this best? (cf. schema/productconfig.rnc in this repo)
+        productschema=...
```

Comment:
@svenseeberg What do I do here?

Reply:
I see 2 options: either set a default path like /usr/share/docserv/config-validation.rnc or make it an configuration option in the docserv.ini.